app/src/org/commcare/android/view/EntityView.java

```diff
@@ -505,8 +539,12 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
             }
         }
 
-        // Re-calculate the view's measurements based on the percentage
-        // adjustments above
+        // Re-calculate the view's measurements based on the percentage adjustments above
         super.onMeasure(widthMeasureSpec, heightMeasureSpec);
+
+        onMeasureCalled = true;
+        if (imageViewsToRedraw.size() > 0) {
+            redrawImageLayoutsInQueue();

```

Comment:
does this need to be called every time `onMeasure` executes, or only the first time? i don't think it makes a difference to the code, just curious for my own understanding.